pkg/cmd/step/nexus/step_nexus.go

```diff
@@ -1,133 +0,0 @@
-package nexus
```

Comment:
So there is no nexus related subcommand in the jx step command. Is that intended? `jx step nexus release` and `jx step nexus drop` were supposed to be removed on Feb 1. I don't see a mention of it in the build pack or jenkins-x-versions repo @hferentschik.

Reply:
No, I guess that's just how the code was written. The actual commands where calling functions in step_nexus (if this is what you mean). 

I am fine with deleting this.